final_assignment/cristina_ungureanu/parser.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# *-* coding: UTF-8 *-*
+
+import os
+import json
+import urllib2
+import subprocess as sp
+import time
+from pprint import pprint
+
+with open('document') as data_file:    
+    data = json.load(data_file)
+
+
+log_file = open("logfile","w")
+log_file.close()
+
+log_file = open("build.log","a")
+
+#before_install
+url_download = data["before_install"][0]["download"]["source"]
+de_descarcat = urllib2.urlopen(url_download)
+fisier_descarcat = open("/home/random/script.sh","w")
+fisier_descarcat.write(de_descarcat.read())
+fisier_descarcat.close()
```

Comment:
Chestie destul de minora, dar e recomandat sa fie in aceeasi limba toate variabilele, mai ales pe open-source stuff.